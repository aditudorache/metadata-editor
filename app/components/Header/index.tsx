import * as React from 'react';

import styled from 'styled-components';
import AppBar from 'components/AppBar';

const StyledHeader = styled.div``;

const Header = () => (
  <StyledHeader>
    <AppBar></AppBar>
  </StyledHeader>
);

export default Header;
