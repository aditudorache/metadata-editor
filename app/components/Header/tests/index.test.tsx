import React from 'react';
import { render } from '@testing-library/react';

import { withAppContext } from 'tests/utils';
import Header from '../index';

describe('<Header />', () => {
  it('should render a div', () => {
    const { container } = render(withAppContext(<Header />));
    expect(container.firstChild).toMatchSnapshot();
  });
});
