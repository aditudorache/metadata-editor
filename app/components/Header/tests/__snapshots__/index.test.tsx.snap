// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> should render a div 1`] = `
<div
  class="sc-AxhCb ccMvAG"
>
  <div
    class="sc-AxiKw hkhXhh"
  >
    <header
      class="MuiPaper-root MuiAppBar-root MuiAppBar-positionStatic MuiAppBar-colorPrimary MuiPaper-elevation4"
    >
      <div
        class="MuiToolbar-root MuiToolbar-regular MuiToolbar-gutters"
      >
        <button
          aria-label="menu"
          class="MuiButtonBase-root MuiIconButton-root sc-AxirZ gkrsXA MuiIconButton-colorInherit MuiIconButton-edgeStart"
          tabindex="0"
          type="button"
        >
          <span
            class="MuiIconButton-label"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root"
              focusable="false"
              role="presentation"
              viewBox="0 0 24 24"
            >
              <path
                d="M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z"
              />
            </svg>
          </span>
          <span
            class="MuiTouchRipple-root"
          />
        </button>
        <h6
          class="MuiTypography-root sc-AxjAm hYgLoi MuiTypography-h6"
        >
          Metadata editor
        </h6>
        <button
          class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-colorInherit"
          tabindex="0"
          type="button"
        >
          <span
            class="MuiButton-label"
          >
            Login
          </span>
          <span
            class="MuiTouchRipple-root"
          />
        </button>
      </div>
    </header>
  </div>
</div>
`;
